os/tools/configure.sh

```diff
@@ -245,4 +245,45 @@ if [ "X${defappdir}" = "Xy" ]; then
     echo "CONFIG_APPS_DIR=\"$posappdir\"" >> "${dest_config}"
   fi
 fi
+
+# Check mandatory package to configuration set
+IOTIVITY=`grep CONFIG_ENABLE_IOTIVITY=y ${dest_config}`
+if [ ! -z ${IOTIVITY} ]; then
+  if ! which scons >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To build ${boardconfig}, scons should be installed"
+    echo "  Find details at https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseIotivity.md"
+    echo " "
+  fi
+fi
+
+ROMFS=`grep CONFIG_FS_ROMFS=y ${dest_config}`
+if [ ! -z ${ROMFS} ]; then
+  if ! which genromfs >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To download ${boardconfig} built image, genromfs should be installed"
+    echo "  Find details at https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseROMFS.md"
+    echo " "
+  fi
+fi
+
+IOTJS=`grep CONFIG_ENABLE_IOTJS=y ${dest_config}`
+if [ ! -z ${IOTJS} ]; then
+  if ! which cmake >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To build ${boardconfig}, cmake should be installed"
+    echo " "
+  fi
+fi
+
+GRPC=`grep CONFIG_PROTOBUF=y ${dest_config}`
+if [ ! -z ${GRPC} ]; then
+  if ! which protoc >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To build ${boardconfig}, protoc should be installed"
+    echo "  Find details at https://github.com/Samsung/TizenRT/blob/master/external/protobuf/README.md"
+    echo " "
+  fi
+fi
+
 echo "  Configuartion is Done!"
```

Comment:
Can we please accommodate below change within this patch?
Configuartion -> Configuration ?

Reply:
Actually that is not in my change. Is it ok to include fixing?

Reply:
Yeah.. It is not in your change. 
Since it is a small change , I thought we can accommodate in this patch. 
If not we can create a new PR.

Reply:
ok. agree that is a small. I will.